### PR TITLE
VOTE-1197: Hatian Creole translations

### PR DESCRIPTION
**Pull request by mlloydbixal**

### CODE REVIEW

File: config/_default/languages.json [hunk 1]
```diff
@@ -238,5 +238,25 @@
...
+      "publishDir": "tmp/public/ht",
+      "register_path": "register",
+      "go_back": "Retounen",
+      "skip_text": "Ale nan kontni prensipal la",
+      "meta__title": "Enskri pou vote",
+      "meta__description": "Jwenn tout opsyon pou enskri pou vote nan Eta pa w.",
+      "english_only": "(ann Anglè)",
+      "last_updated": "Dènye mizajou: ",
+      "select_language": "Chwazi lang",
+      "english_languageName": "Hatian Creole"
```
Comment (rayestrada) [inline config/_default/languages.json:259]: typo

**Review by rayestrada — CHANGES_REQUESTED** [review]
Before I continue review can you make sure that "In English" is showing up for links that are not translated like the other translations?

File: data/translations/ht/homepage.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "banner__guidance_https_text": "Yon kadna %lock_icon% oswa %https% vle di w konekte an tout sekirite sou yon sitwèb .gov. Pataje enfòmasyon sansib sèlman si se sitwèb ofisyèl sekirize yo.",
+  "alert__alert_msg": "COVID-19 kapab lakòz chanjman nan dat, anplasman epi fason w ap vote nan pwochen eleksyon yo. Pou jwenn plis enfòmasyon, verifye sou (sit eleksyon Eta w la).",
+  "alert__alert_after_election": "Pou w jwenn rezilta ofisyèl eleksyon yo, ale sou sitwèb Eta w la.",
+  "header": "Enskri pou vote",
+  "state_selection__heading": "Ki kote w abite?",
+  "state_selection__default": "Chwazi Eta oswa teritwa kote w ap viv la ",
+  "state_selection__submit": "Jwen fason pou enskri",
+  "registered__header": "Ou gentan enskri?",
+  "registered__resources_intro": "Eksplore %link% nou yo sou %USA.gov%",
+  "registered__resources_link_text": "resous sou vote ak eleksyon",
```
Comment (rayestrada) [inline data/translations/ht/homepage.json:16]: Add (in english) translation to link text

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: content/ht/konsènan-nou.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
++++
+title = "Konsènan Vote.gov"
+meta_title = "About Us"
+meta_description = "Vote.gov is a United States government website that provides trusted voting information and can help you find voter registration rules for your state."
```
[the remark this answers is not in the document]
Comment (mlloydbixal) [reply content/ht/konsènan-nou.html:4]: I think we are still waiting on translations for the About Us meta data and the 4th accordion

[the remark this answers is not in the document]
Comment (mlloydbixal) [reply content/ht/konsènan-nou.html]: Found it!

**Review by rayestrada — APPROVED** [review]

File: content/ht/about-us.html [hunk 4]
```diff
@@ -12,8 +12,7 @@ <h2>Misyon</h2>
 
 <h2>Istorik</h2>
 <p>Yon ekip anplwaye USA.gov ak <a href="https://presidentialinnovationfellows.gov/">Presidential Innovation Fellows (ann Anglè)</a> te devlope Vote.gov kòm yon zouti fyab sou entènèt la kote votè ka enskri. Vote.gov te fonde nan lane 2016 pou mete pouvwa enfòmasyon nan men piblik la sou pwosesis vòt ki egzat, ajou, epi pratik.</p>
-
-<p>TJodi a, ekip Vote.gov kontinye amelyore sit entènèt la pou atenn objektif ki tabli nan <a href="https://www.whitehouse.gov/briefing-room/presidential-actions/2021/03/07/executive-order-on-promoting-access-to-voting/">Dekrè Egzekitif Prezidan Biden pou lane 2021 sou Pwomosyon pou Moun Gen Aksè Vote (ann Anglè)</a>. Objektif sa yo enkli amelyore kapasite pou jwenn aksè, tradui zouti votè yo nan lang kle yo, epi amelyore fonksyonalite rechèch sou sitwèb la.</p>
+<p>Jodi a, ekip Vote.gov kontinye amelyore sit entènèt la pou atenn objektif ki tabli nan <a href="https://www.whitehouse.gov/briefing-room/presidential-actions/2021/03/07/executive-order-on-promoting-access-to-voting/">Dekrè Egzekitif Prezidan Biden pou lane 2021 sou Pwomosyon pou Moun Gen Aksè Vote</a>﻿.  </p>
```
Comment (rayestrada) [inline content/ht/about-us.html:15]: Missing last sentence in last paragraph.
```
Objektif sa yo enkli amelyore kapasite pou jwenn aksè, tradui zouti votè yo nan lang kle yo, epi amelyore fonksyonalite rechèch sou sitwèb la.
```

File: data/translations/ht/state-data.json [hunk 5]
```diff
@@ -256,7 +256,7 @@
         "default_mail_postmarked_deadline": "Dwe gen so lapòs 20 jou anvan Jou Eleksyon an"
     },
     "wy": {
-        "default_ip_deadline": "Disponib jiska ki enkli jou Eleksyon an",
+        "default_ip_deadline": "Disponib jiska jou Eleksyon an",
```
Comment (rayestrada) [inline data/translations/ht/state-data.json:259]: This change needs to be done for all `default_ip_deadline`

**Review by rayestrada — CHANGES_REQUESTED** [review]

**Review by rayestrada — DISMISSED** [review]

**Review by rayestrada — APPROVED** [review]

**Comment by rayestrada** [comment]
Resolves #816

**Pull request merged**